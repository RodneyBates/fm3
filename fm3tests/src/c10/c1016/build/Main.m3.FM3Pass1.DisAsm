 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[1,0])
 ItkModuleBeginPatch(C837,L1(16_1),[21,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkLookupScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[22,2])
 ItkStmtListLt(L0(16_0),[22,2])
 ItkBlockDeclListLtPatch(C10,L12(16_c),[5,2])
 ItkBlockDeclListRt(L12(16_c),[5,2])
 ItkBlockDeclListSepPatch(C771,L11(16_b),[19,0])
 ItkConstDeclLtPatch(C771,[19,8])
 ItkConstDeclColonPatch(C777,[19,10],FALSE)
 ItkConstDeclEqualsPatch(C779,[19,10])
 ItkConstDeclRt([19,8])
 ItkDerefLtPatch(C779,L0(16_0),[19,16])
 ItkDerefRt(L0(16_0),[19,16])
 ItkCharLit(L0(16_0),[19,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id13"L",[19,8])
 ItkBlockDeclListSepPatch(C699,L10(16_a),[18,0])
 ItkConstDeclLtPatch(C699,[18,8])
 ItkConstDeclColonPatch(C705,[18,10],FALSE)
 ItkConstDeclEqualsPatch(C707,[18,10])
 ItkConstDeclRt([18,8])
 ItkSubscriptLtPatch(C707,L1(16_1),[18,16])
 ItkSubscriptRt(L1(16_1),[18,16])
 ItkSubscriptsPlusListLtPatch(C712,L1(16_1),[18,16])
 ItkSubscriptsPlusListRt(L1(16_1),[18,16])
 ItkIntLit(L0(16_0),[18,18])
 ItkLongIntLit(L13(16_d),[18,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id17"K",[18,8])
 ItkBlockDeclListSepPatch(C632,L9(16_9),[17,0])
 ItkConstDeclLtPatch(C632,[17,8])
 ItkConstDeclColonPatch(C638,[17,10],FALSE)
 ItkConstDeclEqualsPatch(C640,[17,10])
 ItkConstDeclRt([17,8])
 ItkCallLtPatch(C640,L0(16_0),[17,15])
 ItkCallRt(L0(16_0),[17,15])
 ItkActualsListLtPatch(C645,L0(16_0),[17,15])
 ItkActualsListRt(L0(16_0),[17,15])
 ItkIntLit(L11(16_b),[17,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id16"J",[17,8])
 ItkBlockDeclListSepPatch(C574,L8(16_8),[15,0])
 ItkConstDeclLtPatch(C574,[15,8])
 ItkConstDeclColonPatch(C580,[15,10],FALSE)
 ItkConstDeclEqualsPatch(C582,[15,10])
 ItkConstDeclRt([15,8])
 ItkExprDotLtPatch(C582,Id15"N",[15,18])
 ItkExprDotRt(Id15"N",[15,18])
 ItkQualIdAtoms(Id13"L",Id14"M",[15,12],[15,16])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id12"I",[15,8])
 ItkBlockDeclListSepPatch(C490,L7(16_7),[14,0])
 ItkConstDeclLtPatch(C490,[14,8])
 ItkConstDeclColonPatch(C496,[14,10],FALSE)
 ItkConstDeclEqualsPatch(C498,[14,10])
 ItkConstDeclRt([14,8])
 ItkSubscriptLtPatch(C498,L2(16_2),[14,18])
 ItkSubscriptRt(L2(16_2),[14,18])
 ItkSubscriptsPlusListLtPatch(C503,L2(16_2),[14,18])
 ItkSubscriptsPlusListRt(L2(16_2),[14,18])
 ItkSubscriptsPlusListSepPatch(C508,L1(16_1),[14,22])
 ItkIdRefAtom(Id11"H",[14,24])
 ItkIdRefAtom(Id9"G",[14,20])
 ItkIdRefAtom(Id6"F",[14,14])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id11"H",[14,8])
 ItkBlockDeclListSepPatch(C408,L6(16_6),[13,0])
 ItkConstDeclLtPatch(C408,[13,8])
 ItkConstDeclColonPatch(C414,[13,10],FALSE)
 ItkConstDeclEqualsPatch(C416,[13,10])
 ItkConstDeclRt([13,8])
 ItkCallLtPatch(C416,L1(16_1),[13,18])
 ItkCallRt(L1(16_1),[13,18])
 ItkActualsListLtPatch(C421,L1(16_1),[13,18])
 ItkActualsListRt(L1(16_1),[13,18])
 ItkAnonActualLtPatch(C421,[13,20])
 ItkAnonActualRt([13,20])
 ItkIdRefAtom(Id6"F",[13,20])
 ItkIdRefAtom(Id10"E",[13,14])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id9"G",[13,8])
 ItkBlockDeclListSepPatch(C341,L5(16_5),[12,0])
 ItkConstDeclLtPatch(C341,[12,8])
 ItkConstDeclColonPatch(C347,[12,10],FALSE)
 ItkConstDeclEqualsPatch(C349,[12,10])
 ItkConstDeclRt([12,8])
 ItkCallLtPatch(C349,L0(16_0),[12,18])
 ItkCallRt(L0(16_0),[12,18])
 ItkActualsListLtPatch(C354,L0(16_0),[12,18])
 ItkActualsListRt(L0(16_0),[12,18])
 ItkIdRefAtom(Id5"D",[12,14])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id6"F",[12,8])
 ItkBlockDeclListSepPatch(C286,L4(16_4),[11,0])
 ItkConstDeclLtPatch(C286,[11,8])
 ItkConstDeclColonPatch(C292,[11,10],FALSE)
 ItkConstDeclEqualsPatch(C294,[11,10])
 ItkConstDeclRt([11,8])
 ItkDerefLtPatch(C294,L0(16_0),[11,18])
 ItkDerefRt(L0(16_0),[11,18])
 ItkIdRefAtom(Id4"C",[11,14])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id10"E",[11,8])
 ItkBlockDeclListSepPatch(C202,L3(16_3),[9,0])
 ItkConstDeclLtPatch(C202,[9,8])
 ItkConstDeclColonPatch(C208,[9,10],FALSE)
 ItkConstDeclEqualsPatch(C210,[9,10])
 ItkConstDeclRt([9,8])
 ItkSubscriptLtPatch(C210,L2(16_2),[9,14])
 ItkSubscriptRt(L2(16_2),[9,14])
 ItkSubscriptsPlusListLtPatch(C215,L2(16_2),[9,14])
 ItkSubscriptsPlusListRt(L2(16_2),[9,14])
 ItkSubscriptsPlusListSepPatch(C220,L1(16_1),[9,18])
 ItkIdRefAtom(Id3"B",[9,20])
 ItkIdRefAtom(Id2"A",[9,16])
 ItkIdRefAtom(Id9"G",[9,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id5"D",[9,8])
 ItkBlockDeclListSepPatch(C96,L2(16_2),[8,0])
 ItkConstDeclLtPatch(C96,[8,8])
 ItkConstDeclColonPatch(C102,[8,10],FALSE)
 ItkConstDeclEqualsPatch(C104,[8,10])
 ItkConstDeclRt([8,8])
 ItkCallLtPatch(C104,L2(16_2),[8,14])
 ItkCallRt(L2(16_2),[8,14])
 ItkActualsListLtPatch(C109,L2(16_2),[8,14])
 ItkActualsListRt(L2(16_2),[8,14])
 ItkActualsListSepPatch(C124,L1(16_1),[8,18])
 ItkNamedExprLtPatch(C124,Id3"B",[8,20])
 ItkNamedExprRt(Id3"B",[8,20])
 ItkIdRefAtom(Id8"Y",[8,25])
 ItkAnonActualLtPatch(C109,[8,16])
 ItkAnonActualRt([8,16])
 ItkIdRefAtom(Id7"X",[8,16])
 ItkIdRefAtom(Id6"F",[8,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id4"C",[8,8])
 ItkBlockDeclListSepPatch(C46,L1(16_1),[7,0])
 ItkConstDeclLtPatch(C46,[7,8])
 ItkConstDeclColonPatch(C52,[7,10],FALSE)
 ItkConstDeclEqualsPatch(C54,[7,10])
 ItkConstDeclRt([7,8])
 ItkDerefLtPatch(C54,L0(16_0),[7,14])
 ItkDerefRt(L0(16_0),[7,14])
 ItkIdRefAtom(Id5"D",[7,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id3"B",[7,8])
 ItkConstDeclLtPatch(C10,[5,8])
 ItkConstDeclColonPatch(C16,[5,10],FALSE)
 ItkConstDeclEqualsPatch(C18,[5,10])
 ItkConstDeclRt([5,8])
 ItkQualIdAtoms(Id3"B",Id4"C",[5,12],[5,16])
 ItkDeclTypeAbsent()
 ItkDeclId(L9(16_9),Id2"A",[5,8])
 ItkLookupScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
