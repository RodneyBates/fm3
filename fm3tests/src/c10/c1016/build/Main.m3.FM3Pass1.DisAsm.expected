 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L2(16_2),[1,0])
 ItkModuleBeginPatch(C700,L2(16_2),[15,18])
 ItkModuleRt(L2(16_2),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[15,18])
 ItkStmtListLt(L0(16_0),[15,18])
 ItkOpenDeclListLtPatch(C10,L9(16_9),[5,2])
 ItkOpenDeclListRt(L9(16_9),[5,2])
 ItkConstDeclLtPatch(C644,[15,8])
 ItkConstDeclRt([15,8])
 ItkQualIdAtoms(Id14"L",Id15"M",[15,12],[15,16])
 ItkConstDeclValue([15,10])
 ItkDeclTypeAbsent([15,10])
 ItkConstDeclType([15,10])
 ItkDeclId(L9(16_9),Id13"I",L-9223372036854775808(16_8000000000000000),[15,8])
 ItkOpenDeclListSep(L8(16_8),[5,2])
 ItkConstDeclLtPatch(C551,[14,8])
 ItkConstDeclRt([14,8])
 ItkSubscriptLtPatch(C578,L2(16_2),[14,18])
 ItkSubscriptSubsPatch(C583,L2(16_2),[14,18])
 ItkSubscriptRt(L2(16_2),[14,18])
 ItkSubscriptsPlusListLtPatch(C583,L2(16_2),[14,18])
 ItkSubscriptsPlusListRt(L2(16_2),[14,18])
 ItkIdRefAtom(Id12"H",[14,24])
 ItkSubscriptsPlusListSep(L1(16_1),[14,22])
 ItkIdRefAtom(Id10"G",[14,20])
 ItkIdRefAtom(Id7"F",[14,14])
 ItkConstDeclValue([14,10])
 ItkDeclTypeAbsent([14,10])
 ItkConstDeclType([14,10])
 ItkDeclId(L9(16_9),Id12"H",L-9223372036854775808(16_8000000000000000),[14,8])
 ItkOpenDeclListSep(L7(16_7),[5,2])
 ItkConstDeclLtPatch(C458,[13,8])
 ItkConstDeclRt([13,8])
 ItkCallLtPatch(C485,L1(16_1),[13,18])
 ItkCallActualsPatch(C490,L1(16_1),[13,18])
 ItkCallRt(L1(16_1),[13,18])
 ItkActualsListLtPatch(C490,L1(16_1),[13,18])
 ItkActualsListRt(L1(16_1),[13,18])
 ItkAnonActualLtPatch(C490,[13,20])
 ItkAnonActualRt([13,20])
 ItkIdRefAtom(Id7"F",[13,20])
 ItkIdRefAtom(Id11"E",[13,14])
 ItkConstDeclValue([13,10])
 ItkDeclTypeAbsent([13,10])
 ItkConstDeclType([13,10])
 ItkDeclId(L9(16_9),Id10"G",L-9223372036854775808(16_8000000000000000),[13,8])
 ItkOpenDeclListSep(L6(16_6),[5,2])
 ItkConstDeclLtPatch(C380,[12,8])
 ItkConstDeclRt([12,8])
 ItkCallLtPatch(C407,L0(16_0),[12,18])
 ItkCallActualsPatch(C412,L0(16_0),[12,18])
 ItkCallRt(L0(16_0),[12,18])
 ItkActualsListLtPatch(C412,L0(16_0),[12,18])
 ItkActualsListRt(L0(16_0),[12,18])
 ItkIdRefAtom(Id6"D",[12,14])
 ItkConstDeclValue([12,10])
 ItkDeclTypeAbsent([12,10])
 ItkConstDeclType([12,10])
 ItkDeclId(L9(16_9),Id7"F",L-9223372036854775808(16_8000000000000000),[12,8])
 ItkOpenDeclListSep(L5(16_5),[5,2])
 ItkConstDeclLtPatch(C323,[11,8])
 ItkConstDeclRt([11,8])
 ItkDerefLtPatch(C350,[11,18])
 ItkDerefRt([11,18])
 ItkIdRefAtom(Id5"C",[11,14])
 ItkConstDeclValue([11,10])
 ItkDeclTypeAbsent([11,10])
 ItkConstDeclType([11,10])
 ItkDeclId(L9(16_9),Id11"E",L-9223372036854775808(16_8000000000000000),[11,8])
 ItkOpenDeclListSep(L4(16_4),[5,2])
 ItkConstDeclLtPatch(C230,[9,8])
 ItkConstDeclRt([9,8])
 ItkSubscriptLtPatch(C257,L2(16_2),[9,14])
 ItkSubscriptSubsPatch(C262,L2(16_2),[9,14])
 ItkSubscriptRt(L2(16_2),[9,14])
 ItkSubscriptsPlusListLtPatch(C262,L2(16_2),[9,14])
 ItkSubscriptsPlusListRt(L2(16_2),[9,14])
 ItkIdRefAtom(Id4"B",[9,20])
 ItkSubscriptsPlusListSep(L1(16_1),[9,18])
 ItkIdRefAtom(Id3"A",[9,16])
 ItkIdRefAtom(Id10"G",[9,12])
 ItkConstDeclValue([9,10])
 ItkDeclTypeAbsent([9,10])
 ItkConstDeclType([9,10])
 ItkDeclId(L9(16_9),Id6"D",L-9223372036854775808(16_8000000000000000),[9,8])
 ItkOpenDeclListSep(L3(16_3),[5,2])
 ItkConstDeclLtPatch(C115,[8,8])
 ItkConstDeclRt([8,8])
 ItkCallLtPatch(C142,L2(16_2),[8,14])
 ItkCallActualsPatch(C147,L2(16_2),[8,14])
 ItkCallRt(L2(16_2),[8,14])
 ItkActualsListLtPatch(C147,L2(16_2),[8,14])
 ItkActualsListRt(L2(16_2),[8,14])
 ItkNamedExprLtPatch(C167,Id4"B",[8,20])
 ItkNamedExprRt(Id4"B",[8,20])
 ItkIdRefAtom(Id9"Y",[8,25])
 ItkActualsListSep(L1(16_1),[8,18])
 ItkAnonActualLtPatch(C147,[8,16])
 ItkAnonActualRt([8,16])
 ItkIdRefAtom(Id8"X",[8,16])
 ItkIdRefAtom(Id7"F",[8,12])
 ItkConstDeclValue([8,10])
 ItkDeclTypeAbsent([8,10])
 ItkConstDeclType([8,10])
 ItkDeclId(L9(16_9),Id5"C",L-9223372036854775808(16_8000000000000000),[8,8])
 ItkOpenDeclListSep(L2(16_2),[5,2])
 ItkConstDeclLtPatch(C59,[7,8])
 ItkConstDeclRt([7,8])
 ItkDerefLtPatch(C86,[7,14])
 ItkDerefRt([7,14])
 ItkIdRefAtom(Id6"D",[7,12])
 ItkConstDeclValue([7,10])
 ItkDeclTypeAbsent([7,10])
 ItkConstDeclType([7,10])
 ItkDeclId(L9(16_9),Id4"B",L-9223372036854775808(16_8000000000000000),[7,8])
 ItkOpenDeclListSep(L1(16_1),[5,2])
 ItkConstDeclLtPatch(C10,[5,8])
 ItkConstDeclRt([5,8])
 ItkQualIdAtoms(Id4"B",Id5"C",[5,12],[5,16])
 ItkConstDeclValue([5,10])
 ItkDeclTypeAbsent([5,10])
 ItkConstDeclType([5,10])
 ItkDeclId(L9(16_9),Id3"A",L-9223372036854775808(16_8000000000000000),[5,8])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
